charts/newrelic-infrastructure/templates/kubelet/daemonset.yaml

```diff
@@ -16,41 +17,42 @@ spec:
   {{- end }}
   selector:
     matchLabels:
-      {{- include "common.labels.selectorLabels" . | nindent 6 }}
+      {{- include "newrelic.common.labels.selectorLabels" . | nindent 6 }}
       app.kubernetes.io/component: kubelet
   template:
     metadata:
       annotations:
         checksum/nri-kubernetes: {{ include (print $.Template.BasePath "/kubelet/scraper-configmap.yaml") . | sha256sum }}
         checksum/agent-config: {{ include (print $.Template.BasePath "/kubelet/agent-configmap.yaml") . | sha256sum }}
-        {{- if include "common.license.secret" . }}{{- /* If the is secret to template */}}
+        {{- if include "newrelic.common.license.secret" . }}{{- /* If the is secret to template */}}
         checksum/license-secret: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
         {{- end }}
         checksum/integrations_config: {{ include (print $.Template.BasePath "/kubelet/integrations-configmap.yaml") . | sha256sum }}
         {{- with .Values.podAnnotations }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
       labels:
-        {{- include "common.labels.podLabels" . | nindent 8 }}
+        {{- include "newrelic.common.labels.podLabels" . | nindent 8 }}
         app.kubernetes.io/component: kubelet
+        mode: {{ if not ( include "newrelic.common.privileged" . ) }}un{{ end }}privileged
     spec:
-      {{- with include "common.images.renderPullSecrets" ( dict "pullSecrets" (list .Values.images.pullSecrets) "context" .) }}
+      {{- with include "newrelic.common.images.renderPullSecrets" ( dict "pullSecrets" (list .Values.images.pullSecrets) "context" .) }}
       imagePullSecrets:
         {{- . | nindent 8 }}
       {{- end }}
-      {{- with include "common.dnsConfig" . }}
+      {{- with include "newrelic.common.dnsConfig" . }}
       dnsConfig:
         {{- . | nindent 8 }}
       {{- end }}
-      {{- with include "common.priorityClassName" . }}
+      {{- with include "newrelic.common.priorityClassName" . }}
       priorityClassName: {{ . }}
       {{- end }}
-      {{- with include "common.securityContext.pod" . }}
+      {{- with include "newrelic.common.securityContext.pod" . }}
```

Comment:
This had an open issue in the previous PR that may need to be taken into consideration again: https://github.com/newrelic/nri-kubernetes/pull/383#discussion_r844178708

Reply:
It was before but the overriding system left it a bit hidden.